apps/miniapp/global.d.ts

```diff
@@ -0,0 +1,24 @@
+import enCondo from '@app/condo/lang/en/en.json'
+import ruCondo from '@app/condo/lang/ru/ru.json'
+import en from '@app/miniapp/lang/en.json'
+import ru from '@app/miniapp/lang/ru.json'
+
+
+// NOTE: Combine all keys together
+const translations = [
+    en, ru,
+]
+const translationsCondo = [
+    enCondo, ruCondo,
+]
+
+export type MessagesKeysType = keyof typeof translations[number] | keyof typeof translationsCondo[number]
```

Comment:
👀 was it always like that? I thought each miniapp should have its own set of translations

Reply:
Mb TODO here?